numpy/core/fromnumeric.py

```diff
@@ -2343,11 +2362,21 @@ def amax(a, axis=None, out=None, keepdims=np._NoValue):
     >>> np.nanmax(b)
     4.0
 
+    You can use an initializer to compute the maximum of an empty slice, or
+    to initialize it to a different value:
+
+    >>> np.max([], initializer=-np.inf)
+    -inf
+    >>> np.max([5], initializer=6)
+    6
```

Comment:
Comparison to the python 3 `max(default=...)` would be good here. Same for `min`